client/app/cloud/project/compute/loadbalancer/cloud-project-compute-loadbalancer.controller.js

```diff
@@ -0,0 +1,98 @@
+"use strict";
+
+angular.module("managerApp")
+  .controller("CloudProjectComputeLoadbalancerCtrl", function ($q, $scope, $translate, CloudProjectComputeLoadbalancerService, OvhApiCloudProjectIplb, OvhApiIpLoadBalancing, ControllerHelper, CloudMessage, $stateParams, OvhApiMe, URLS) {
+
+    var self = this,
```

Comment:
use let AND separate lines for variable declaration

Reply:
done

Reply:
Where? can you verify you did push it?